docs/light_configuration.md

```
@@ -68,7 +70,103 @@ You can also **offset** solar times:
+Examples:
+
+```yacas
+smooth(sunrise, 14d)
+clamp(smooth(sunrise, 14d), 06:30, 08:00)
+```
+</details>
+
+<details>
+<summary>Examples</summary>
+
+~~~
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)